Documentation/operations/performance/tuning.rst

```diff
@@ -178,7 +178,9 @@ To enable the iptables connection-tracking bypass:
 
        .. code-block:: shell-session
 
-          cilium install --config install-no-conntrack-iptables-rules=true
+          cilium install \\
+            --set installNoConntrackIptablesRules=true \\
+            --set kubeProxyReplacement=true
```

Comment:
When in a code-block (differently from a parsed-literal block as below), `\` does not need to be escaped.